.travis.yml

```diff
@@ -8,47 +8,27 @@ before_install:
   - gem install bundler
 
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.9
-  - 2.2.6
-  - 2.3.3
-  - 2.4.0
+  - 2.4.6
+  - 2.5.5
+  - 2.6.3
 
 gemfile:
   - Gemfile
-  - gemfiles/Gemfile-4-0-stable
-  - gemfiles/Gemfile-4-1-stable
-  - gemfiles/Gemfile-4-2-stable
   - gemfiles/Gemfile-5-0-stable
+  - gemfiles/Gemfile-5-1-stable
+  - gemfiles/Gemfile-5-2-stable
   - gemfiles/Gemfile-edge
 
 matrix:
   allow_failures:
     - gemfile: gemfiles/Gemfile-edge
+    - gemfile: gemfiles/Gemfile-6-0-stable
```

Comment:
Is it failing right now? Because if it is not we should not allow failures. A RC is basically the final release so it should be passing.

Reply:
Yep, I've left it in the test matrix and its failing on Rails 6 stable.

Reply:
👍. I'll merge as is and we can fix the broken test on 6-0-stable in a next PR.

Reply:
Thanks!  Do you know if anything has changed between 5.2 and 6.0 in terms of xml/HTTP_ACCEPT ?

Reply:
I think the template lookup changed and maybe affected this.

Reply:
@rafaelfranca I've investigated and PR'ed a fix https://github.com/rails/actionpack-page_caching/pull/57